### PR TITLE
DB-9028 Fix ClassCastException when query contains round()

### PR DESCRIPTION
**Pull request by changli6**

### CODE REVIEW

**Comment by changli6** [comment]
jenkins please teste branch @cdh5.12.2

**Comment by changli6** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST FAILED

**Comment by changli6** [comment]
jenkins please test branch @cdh5.12.2

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/OperatorToString.java [hunk 1]
```diff
@@ -968,7 +968,7 @@ else if (operand instanceof ConditionalNode) {
                             methodName.equals("ADD_MONTHS")    ||
                             methodName.equals("ADD_YEARS")     ||
                             methodName.equals("ADD_DAYS")      ||
-                            methodName.equals("ROUND"))
+                            methodName.equalsIgnoreCase("ROUND"))
```
Comment (arnaud-splice) [inline db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/OperatorToString.java]: Shouldn't that apply to every other item in this list?

Comment (changli6) [reply db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/OperatorToString.java]: Won't need to do that. Other functions were "made in splice" and their method names were uppercase. The round function came from Splice package (uppercase) and java.lang.StrictMath (lowercase), we need to handle this one especially.

Comment (arnaud-splice) [reply db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/OperatorToString.java]: All right! Could you add a comment in the code to explain that?

Comment (changli6) [reply db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/OperatorToString.java]: Done

**Pull request merged**